src/util/Constants.js

```diff
@@ -205,6 +205,7 @@ const Endpoints = exports.Endpoints = {
     return {
       Emoji: emojiID => `${root}/emojis/${emojiID}.png`,
       Asset: name => `${root}/assets/${name}`,
+      DefaultAvatar: number => `${root}/embed/avatars/${number}.png`,
```

Comment:
After checking the links you have shown, yes they do work. However, this contradicts the answer to my original quiestion, as default avatar url links used to be like: https://discordapp.com/assets/0e291f67c9274a1abdddeb3fd919cbaa.png